### DISCUSSION TITLE
Downloads

### DISCUSSION THREAD

**Discussion by Jose-jme** [zen-browser/desktop#1960] [Ideas]
Hi, I'm a Linux user and I've been racking my brains trying to install the browser on Ubuntu in a way that isn't flatpak. It would be very comforting to have it in a .deb file and it would also make it available to more users.

**Comment by homeschoolu** [comment]
I've read good things about this browser. I would like to try it.
I've used Firefox for years. Tried others, but prefer Firefox.
I run Linux since mid-1990's, coming up thru Ubuntu and derivatives to Debian and derivatives.
I've now run MX Linux for a few years and use the .deb.packages. No flat, no snap, no app by old time choice.
.. Well, lots of work you have done. Looks interesting and best of luck to you. I'll check back. Thanks.

**Reply by homeschoolu** [reply]
Just discovered: 
https://github.com/GitXpresso/Browsers-NoVNC/releases/download/TarAndDeb/zen.linux-x86_64.deb
